src/Compilers/CSharp/Portable/Binder/Binder_Statements.cs

```diff
@@ -2994,23 +2994,23 @@ private void ReportCantConvertLambdaReturn(SyntaxNode syntax, DiagnosticBag diag
                         lambda.MessageID.Localize());
                 }
             }
+        }
 
-            Location getLocationForDiagnostics(SyntaxNode node)
+        private static Location GetLocationForDiagnostics(SyntaxNode node)
```

Comment:
>GetLocationForDiagnostics [](start = 32, length = 25)

looks like this is still only used in one method. Could still be a local function